### PR TITLE
Er 1199/workflow transition e2e

### PR DESCRIPTION
**Pull request by bleto**
Workflow creates transition e2e tests and refactor old workflow tests

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [x] e2e Test

# Checklist:

- [x] I have read the contribution requirements and fulfil them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: cypress/integration/11:workflow/transitions/1:create.feature [hunk 1]
```diff
@@ -10,3 +10,18 @@ Feature: CREATE: Transition on workflow
...
+    * On "workflow-designer" I can see column with "status_EN" value
+
+  @other
+  Scenario: Add workflow transition - give up
+    When On "workflow-designer" in row 2 I create transition from "New" to "status_EN"
+    * On "workflow-designer" I click on the "remove" button for the transition from "New" to "status_EN"
+    Then I confirm modal
+
+  @success
+  Scenario: Add workflow transition - success
```
Comment (derpdead) [inline cypress/integration/11:workflow/transitions/1:create.feature:22]: Creating multiple transitions? From New => status_EN and from Draft => New ? Isn't one transition enough?

File: cypress/integration/11:workflow/transitions/1:create/index.test.js [hunk 2]
```diff
@@ -57,38 +52,47 @@ MultiSteps([
...
         .find('.workflow-designer-header-layer-cell__title span')
-        .contains(new RegExp(`${escapeStringRegexp(searchValue)}$`, 'g'));
+        .each(($el) => {
+            statuses.push($el.text().trim());
+        }).then(() => {
+            const fromIndex = statuses.findIndex(status => status === from);
+            const toIndex = statuses.findIndex(status => status === to);
+
+            cy
+                .get('@grid')
```
Comment (derpdead) [inline cypress/integration/11:workflow/transitions/1:create/index.test.js:86]: Code duplication? Doing same action twice?

File: cypress/integration/common/actions.test.js [hunk 3]
```diff
@@ -145,3 +146,68 @@ MultiSteps([
...
+    Then,
+    When,
+], 'On {string} I click on {string} button for column with {string} value', (gridId, action, searchValue) => {
+    cy
+        .get(`[data-cy=${gridId}]`)
+        .should('be.visible')
+        .as('grid');
+    cy
+        .get('@grid')
+        .find('.workflow-designer-header-layer-cell')
```
Comment (derpdead) [inline cypress/integration/common/actions.test.js:162]: I am thinking if it's really necessary to do multiple finding elements. Gotta check what is really faster for cypress - single find, or multiple.

File: cypress/support/commands.js [hunk 4]
```diff
@@ -87,3 +87,36 @@ Cypress.on('uncaught:exception', (err) => {
...
+});
+
+Cypress.Commands.add('openPage', ({
+    page, requestAlias,
+}) => {
+    cy.visit(`/${page}`);
+    cy
+        .url()
+        .should('include', `/${page}`);
+    cy.checkRequest({
```
Comment (derpdead) [inline cypress/support/commands.js:118]: I would suggest to not mix a context => Open page !== check request. Also after opening page there might be multiple requests going on

**Review by derpdead — CHANGES_REQUESTED** [review]
Comments

File: cypress/integration/11:workflow/transitions/1:create.feature (hunk 1 above)
Comment (bleto) [reply cypress/integration/11:workflow/transitions/1:create.feature:22]: yes, one enough

File: cypress/integration/11:workflow/transitions/1:create/index.test.js (hunk 2 above)
Comment (bleto) [reply cypress/integration/11:workflow/transitions/1:create/index.test.js:86]: save `@grid` means that this is an alias for an already saved item , but I'll make it even simpler.

File: cypress/support/commands.js (hunk 4 above)
Comment (bleto) [reply cypress/support/commands.js:118]: ok I will edit the `openPage` method to be prepared to accept multiple requests

**Review by derpdead — APPROVED** [review]

**Pull request merged**